File: Maths_And_Stats/Statistics/Mean/mean.py

#Taking Input
print("Enter the size of list : ")
size = int(input())

print("Enter the list elements: ")
arr = list(map(int,input().split()))

total_sum = sum(arr)

mean = total_sum /size

print("Mean is : {}".format(mean))

'''
    Sample Input :
        enter the size of list: 5
        Enter the list elements : 10 30 20 40 50
    
    Sample Output:
        Mean is 30.0
'''

File: Maths_And_Stats/Statistics/Mean/mean.java
package Maths_And_Stats.Statistics.Mean;


public class mean {

   public static void main(String args[]){
      float mean;
      int sum, i;
      int n = 5;
      int a[] = {2,6,7,4,9};
      sum = 0;

      for(i = 0; i < n; i++) {
         sum+=a[i];
      }
      System.out.println("Mean ::"+ sum/(float)n);
   }
}